tests/test_models.py

```
@@ -14,6 +14,8 @@
 from dreem.models.transformer import (
     TransformerEncoderLayer,
     TransformerDecoderLayer,
+    spatial_emb_from_bb,
+    apply_embeddings,
```

Ruff

`dreem.models.transformer.apply_embeddings` imported but unused

Remove unused import: `dreem.models.transformer.apply_embeddings`

(F401)